pkg/assets/handlers.go

```diff
@@ -141,6 +144,18 @@ func HTML5ModeHandler(contextRoot string, subcontextMap map[string]string, h htt
 	}), nil
 }
 
+var versionTemplate = template.Must(template.New("webConsoleVersion").Parse(`
+window.OPENSHIFT_VERSION = {
+  openshift: "{{ .OpenShiftVersionInfo.GitVersion | js}}",
+  kubernetes: "{{ .KubernetesVersionInfo.GitVersion | js}}"
+};
+`))
+
+type WebConsoleVersion struct {
+	KubernetesVersionInfo kversion.Info
+	OpenShiftVersionInfo  oversion.Info
```

Comment:
don't pass the whole version struct to the template, just put the version string in the struct given to the template. that removes the need to make this package depend on the version packages, and gives us compiler checks on the use of the GitVersion field